astropy/units/tests/test_equivalencies.py

```diff
@@ -681,8 +681,10 @@ def test_temperature():
 def test_temperature_energy():
     x = 1000 * u.K
     y = (x * constants.k_B).to(u.keV)
-    assert_allclose(x.to_value(u.keV, u.temperature_energy()), y.value)
-    assert_allclose(y.to_value(u.K, u.temperature_energy()), x.value)
+    assert_allclose(
+        x.to_value(u.keV, u.temperature_energy()), y.value, rtol=1e-6)
```

Comment:
I think this is because in CODATA2018, we now mathematically derive some of the constants in real time instead of pre-calculating it "on paper" and hardcoding the answer.